### PR TITLE
Community Page updates June 7 , 2021

### PR DESCRIPTION
**Pull request by rfustino**

### CODE REVIEW

File: docs/community.md [hunk 1]
```diff
@@ -78,7 +84,7 @@ If you have a project you want featured on the community site, [reach out](mailt
 |------|------|
 | <center>[MessagePack Debugger](https://github.com/algorand/messagepack-debugger)</center> | View and compare messagepack encoded objects. |
 | <center>[PyTeal](https://github.com/algorand/pyteal)</center> | PyTeal is a Python language binding program for ASC1. PyTeal allows developer to express TEAL-based smart contract logic using Python. This dev tool abstracts TEAL and does type-checking at contraction time.|
-| <center>[Reach](https://reach.sh)</center> | Write complex smart contracts in a JS-like language. Currently does not target MainNet / TestNet, but you can already test locally with ease. See blog update [here](https://medium.com/reach-sh/reach-september-update-7d2bb04bcd7e).  |
+| <center>[Reach](https://reach.sh)</center> | Write complex smart contracts in a JS-like language. Currently does not target MainNet / TestNet, but you can already test locally with ease. See docs [here](https://docs.reach.sh/).  |
```
Comment (ryanRfox) [inline docs/community.md:87]: You can target MainNet and TestNet at this time. Reword this to something like:
 
Write complex smart contracts in a JS-like language. Deploy to MainNet, TestNet or build and test locally with ease. See docs [here](https://docs.reach.sh/).

Comment (rfustino) [reply docs/community.md]: updated. Nice catch.

File: docs/community.md [hunk 2]
```diff
@@ -1,102 +1,113 @@
...
+| <center>[How big is an Algorand Archival Node?](https://howbigisalgorand.com/)</center>              | Dashboard about the size of an Algorand Archival Node. Data is live. Thanks [Harsh Baid](https://forum.algorand.org/u/harshbaid/summary)!                                                                                                                                                                                                                                                                                                                                                                                                |     | <center>[Sandbox](https://github.com/algorand/sandbox)</center> | Algorand Sandbox. |
```
Comment (ryanRfox) [inline docs/community.md:81]: Remove "Thanks [Harsh Baid](https://forum.algorand.org/u/harshbaid/summary)!" (looks like a comment, not description)

**Review by ryanRfox — CHANGES_REQUESTED** [review]

File: docs/community.md (hunk 2 above)
Comment (rfustino) [reply docs/community.md]: This is resolved in this commit: https://github.com/algorand/docs/pull/423/commits/2a9cad155e6eca0b606cae8f108e6d631db6d45d

**Review by ryanRfox — APPROVED** [review]
LGTM

File: docs/community.md [hunk 3]
```diff
@@ -43,9 +43,14 @@ If you have a project you want featured on the community site, [reach out](mailt
 | Name| Description |
 |------|------|
 | <center>[.NET SDK](https://github.com/RileyGe/dotnet-algorand-sdk)</center> | This libary is built with C# and provides Algorand 2.0 features and is compliant with .NET Standard 2.0.
-| <center>[Algoduino](https://algoduino.com/)</center> | SDK for the Internet of Things. A simple easy to use API client implementation in C++ to interact with the Algorand Blockchain.
-| <center>[Dart SDK](https://github.com/alvistar/dart_algorand)</center> | SDK for Dart to interact with the Algorand Blockchain. This SDK can be used to create [Flutter apps](https://github.com/alvistar/algorand_flutter).|
-| <center>[Rust SDK](https://github.com/KBryan/algorand_rust_sdk) | RUST implementation of Algorand's REST API.|
+| <center>[Algoduino](https://algoduino.com/)</center> | SDK for the Internet of Things. A simple easy to use API client implementation in C++ to interact with the Algorand Blockchain. Solution [here](https://developer.algorand.org/solutions/arduino-algorand-blockchain/).
```
Comment (TedNIVAN) [inline docs/community.md]: Algoduino is now obsolete. It has been replaced by Vertices Network.

File: docs/community.md [hunk 4]
```diff
@@ -43,9 +43,14 @@ If you have a project you want featured on the community site, [reach out](mailt
...
+| <center>[Rust SDK (1)](https://github.com/KBryan/algorand_rust_sdk) | RUST implementation of Algorand's REST API.|
+| <center>[Rust SDK (2)](https://github.com/manuelmauro/algonaut) | Rust algonaut aims at becoming a rusty SDK for Algorand.|
+| <center>[Swift SDK](https://github.com/Jesulonimi21/Swift-Algorand-Sdk) | This is a swift sdk that allows interaction with the Algorand blockchain. It also supports interecting with the V2 indexer and Algo APIs. Tutorial [here](https://developer.algorand.org/tutorials/developing-ios-apps-using-swift-sdk/). And Solution [here](https://developer.algorand.org/solutions/building-ios-apps-using-swift-algorand-sdk/)|
+| <center>[Verticies SDK](https://vertices.network/) | Build secure and advanced IoT applications using Algorand blockchain technology. SDK quickstart [here](https://docs.vertices.network/vertices-sdk/quickstart). PanoramiQ and Smart Contract Toolbox coming soon.|
```
Comment (TedNIVAN) [inline docs/community.md]: Please add this description instead:

```
| <center>[Vertices SDK](https://docs.vertices.network/vertices-sdk/quickstart) | C SDK to interact with the Algorand Blockchain. Designed for the Internet of Things. |
```

**Review by TedNIVAN — CHANGES_REQUESTED** [review]
I've added comments regarding Algoduino and Vertices Network.

**Comment by rfustino** [comment]
This PR is outdated, use this updated PR here: https://github.com/algorand/docs/pull/450

**Comment by rfustino** [comment]
This PR is outdated, use this updated PR here: #450
